# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/abiword/hotfix-libxml.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- abiword-3.0.5/src/af/util/xp/ut_stringbuf.cpp.vanilla	2024-05-08 19:25:46.506000000 +0200
+++ abiword-3.0.5/src/af/util/xp/ut_stringbuf.cpp	2024-05-08 19:25:57.728000000 +0200
@@ -26,6 +26,7 @@
 #include <algorithm>
 
 #include <libxml/uri.h>
+#include <libxml/xmlmemory.h>
 
 #include <glib.h>
 
